code/modules/projectiles/guns/smgs.dm

```diff
@@ -190,7 +190,7 @@
 	accuracy_mult = 1.1
 	accuracy_mult_unwielded = 0.95
 	scatter = 4
-	fire_delay = 0.2 SECONDS
+	fire_delay = 0.18 SECONDS
```

Comment:
```suggestion
	fire_delay = 0.18 SECONDS
```
This is rounded up to my knowledge, even if you were making it fire fast I'd rather recommend you remove the scatter entirely and give it more attachment options, the entire gimmick of the MP5 is SMG stats with rifle fire rate and attachment options.

Reply:
I didn't know that was its gimmick. I didn't even know it had one, I thought someone thought it was fun to add it, and there we got.  I'll see about making some changes this week.